frontend/src/component/admin/groups/GroupsList/GroupCard/GroupCard.tsx

```diff
@@ -85,6 +87,10 @@ const InfoBadgeDescription = styled('span')(({ theme }) => ({
     fontSize: theme.fontSizes.smallBody,
 }));
 
+const ProjectNameBadge = styled(Badge)({
+    wordwrap: 'anywhere',
```

Comment:
I haven't seen this anywhere else in our codebase. How would it work with 'break-word'?

Reply:
looks like styled has some issues with the word-wrap property (`wordWrap` would render correctly as `word-wrap`, but linter told me the property was invalid; `wordwrap` was accepted by the linter, but rendered as `wordwrap` so it didn't work), so I'll go with `word-break: break-word` instead.